code/modules/modular_armor/attachables.dm

```diff
@@ -34,11 +34,11 @@
 /** Shoulder lamp strength module */
 /obj/item/armor_module/attachable/better_shoulder_lamp
 	name = "\improper Mark 2 Baldur Light Amplification System"
-	desc = "Designed for mounting on the Jaeger Combat Exoskeleton. Substantially increases the power output of the Jaeger Combat Exoskeleton's mounted flashlight. Slows you down minorly."
+	desc = "Designed for mounting on the Jaeger Combat Exoskeleton. Substantially increases the power output of the Jaeger Combat Exoskeleton's mounted flashlight."
 	icon_state = "mod_lamp_icon"
 	item_state = "mod_lamp"
 	slowdown = 0
-	var/power_boost = 5 /// The boost to armor shoulder light
+	var/power_boost = 7 /// The boost to armor shoulder light
```

Comment:
have you tested if this does anything ? with new lights, there's a serious possibility this doesn't do anything

Reply:
I tested after i changed it, gonna test pre change and after with a measure.